drivers/CoreIMU/include/CoreLSM6DSOX.h

```diff
@@ -0,0 +1,86 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "drivers/I2C.h"
+
+#include "lsm6dsox_reg.h"
+
+namespace leka {
+
+class CoreLSM6DSOX
+{
+  public:
+	explicit CoreLSM6DSOX(mbed::I2C &i2c);
+
+	enum class RateXL
```

Comment:
I would move the rates and ranges outside the class and make them public. Then use a switch case to retrieve the food value 

When we add the magnetometer it will be better to have these kind of things publicly accessible.